### DISCUSSION TITLE
Add custom fonts

### DISCUSSION THREAD

**Discussion by Quin452** [uxbox/uxbox#577] [Ideas]
I am so excited for an official release of the app; I'm already a huge fan (in case it wasn't clear on here or Twitter).

I've just been looking through the Typography section, and noticed there is only "local" and Google fonts listed.
Now, as a rule of thumb, I don't rely on externally hosted assets for my projects, so my sites will not use Google Fonts.  I generally use Bitter and Quicksand as my "default" font family, as well as FontAwesome as my icon font.

I'm just wondering if there will be an option to upload our own fonts (in case they're not a part of Google Fonts), and/or implement icon fonts like FontAwesome.

((I did a search for "fonts", so I assume no one has asked this yet 😛))

**Comment by myfunnyandy** [comment]
Hi, loving your enthusiasm!

Currently you can not import local fonts, but we have planned a lot of improvements regarding font management, and importing local fonts is among them. I will add the suggestion for icon fonts to consider it once we approach the definition of the story.

Here is the Taiga user story: [https://tree.taiga.io/project/penpot/us/292](https://tree.taiga.io/project/penpot/us/292)

**Comment by ruihildt** [comment]
Just wanted to add my voice and enthusiasm to this feature request.

For any specific project not using Google Fonts (and there are a lot^^), this is needed to be able to create a quality mockup.

**Reply by niwinz** [reply]
We already working on it ;)
